src/amo/css/OverallRating.scss

```diff
@@ -0,0 +1,40 @@
+.OverallRating {
+
+  form {
+    display: flex;
+    justify-content: center;
+
+    input {
+      appearance: none;
+      opacity: 0;
+      width: 0;
+      height: 0;
+    }
+
+    label {
```

Comment:
We don't have a specific CSS style guide but I've been trying to keep things as single class names if possible. I think in this case the style engine would need to check each `<label>` to see if it's in a `<form>` and a `.OverallRating`. Probably isn't a big performance hit but if we keep using bare element selectors we could end up with a bunch of that.

I'd prefer:

```scss
.OverallRating-form { }
.OverallRating-input { }
.OverallRating-label { }
.OverallRating-label-love-it { }
// ...
```

It's a bit weird and I'm not sure I'd suggest it but it looks like you can omit the common class portion with:

```scss
.OverallRating {
  &-form { }
  &-input { }
}